Crash reporst 2.9 - ClassCastException on account creation

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.accounts.NewAccountActivity}: android.view.InflateException: Binary XML file line #8: Error inflating class fragment
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2097)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2122)
       at android.app.ActivityThread.access$600(ActivityThread.java:140)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1228)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4895)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:994)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:761)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.view.InflateException: Binary XML file line #8: Error inflating class fragment
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
       at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:306)
       at android.app.Activity.setContentView(Activity.java:1912)
       at org.wordpress.android.ui.accounts.NewAccountActivity.onCreate(NewAccountActivity.java:15)
       at android.app.Activity.performCreate(Activity.java:5163)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2061)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2122)
       at android.app.ActivityThread.access$600(ActivityThread.java:140)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1228)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4895)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:994)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:761)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.ClassCastException: org.wordpress.android.ui.accounts.NewUserPageFragment cannot be cast to android.app.Fragment
       at android.app.Fragment.instantiate(Fragment.java:577)
       at android.app.Fragment.instantiate(Fragment.java:552)
       at android.app.Activity.onCreateView(Activity.java:4806)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:680)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
       at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:306)
       at android.app.Activity.setContentView(Activity.java:1912)
       at org.wordpress.android.ui.accounts.NewAccountActivity.onCreate(NewAccountActivity.java:15)
       at android.app.Activity.performCreate(Activity.java:5163)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2061)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2122)
       at android.app.ActivityThread.access$600(ActivityThread.java:140)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1228)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4895)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:994)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:761)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/src/org/wordpress/android/ui/accounts/NewAccountActivity.java
+++ b/src/org/wordpress/android/ui/accounts/NewAccountActivity.java
@@ -1,13 +1,13 @@
 package org.wordpress.android.ui.accounts;
 
-import android.app.Activity;
 import android.os.Bundle;
+import android.support.v4.app.FragmentActivity;
 import android.view.Window;
 
 import org.wordpress.android.R;
 
 // TODO: merge it with WelcomeFragmentSignIn
-public class NewAccountActivity extends Activity {
+public class NewAccountActivity extends FragmentActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);

